docs/reference/extensions/spring-boot-framework.rst

```diff
@@ -0,0 +1,112 @@
+.. _spring-boot-framework-reference:
+
+spring-boot-framework
```

Comment:
I see [Spring Boot](https://spring.io/projects/spring-boot) and [Spring Framework](https://spring.io/projects/spring-framework), but no Sprint Boot Framework. Which does this correspond to?

Reply:
It corresponds to Spring Boot; the `-framework` here refers to the convention in rockcraft (and charmcraft) of ending the 12f extensions with `-framework`. Same as `django-framework`, `fastapi-framework`, etc
so it's an unfortunate overload.